Use MKDIR_P instead of mkdir_p for compability with modern autotools wrt #477732

--- configure.ac
+++ configure.ac
@@ -74,6 +74,8 @@
 dnl Checks for programs.
 AC_AIX
 
+AC_PROG_MKDIR_P
+
 if test "x$GCC" != "xyes" 
 then
   AC_MSG_WARN([
--- doc/ja/Makefile.am
+++ doc/ja/Makefile.am
@@ -25,7 +25,7 @@
 
 install-man1: $(man_MANS)
 	@$(NORMAL_INSTALL)
-	test -z "$(man1dir)" || $(mkdir_p) "$(DESTDIR)$(mandir)$(mansubdir)"
+	test -z "$(man1dir)" || $(MKDIR_P) "$(DESTDIR)$(mandir)$(mansubdir)"
 	@list='$(man1_MANS)'; \
 	l2='$(man_MANS)'; for i in $$l2; do \
 	  case "$$i" in \
